src/views/Home/AlertsTable/index.tsx

```diff
@@ -218,27 +202,29 @@ function AlertsTable() {
     return (
         <Container
             className={styles.alertsTable}
+            childrenContainerClassName={styles.mainContent}
             heading={strings.allOngoingAlertTitle}
             withHeaderBorder
             withGridViewInFilter
             footerActions={(
                 <Pager
                     activePage={page}
                     itemsCount={data?.count ?? 0}
-                    maxItemsPerPage={limit}
+                    maxItemsPerPage={PAGE_SIZE}
                     onActivePageChange={setPage}
                 />
             )}
+            empty={data?.items?.length === 0}
+            errored={isDefined(alertInfosError)}
+            filtered={false}
         >
-            <SortContext.Provider value={sortState}>
-                <Table
-                    pending={loading}
-                    filtered={filtered}
-                    columns={columns}
-                    keySelector={alertKeySelector}
-                    data={data?.items}
-                />
-            </SortContext.Provider>
+            <Table
+                pending={alertInfoLoading}
+                filtered={false}
```

Comment:
filtered, errored state should be handled since we are passing down the `alertsFilter` and using it.